packages/dev/core/src/Engines/Extensions/engine.renderTargetCube.ts

```diff
@@ -14,11 +14,11 @@ declare module "../../Engines/thinEngine" {
          * @param options defines the options used to create the texture
          * @returns a new render target cube wrapper
          */
-        createRenderTargetCubeTexture(size: number, options?: Partial<RenderTargetCreationOptions>): RenderTargetWrapper;
+        createRenderTargetCubeTexture(size: number, options?: RenderTargetCreationOptions): RenderTargetWrapper;

```

Comment:
`RenderTargetCreationOptions` is already fully partial, so putting `Partial<>` on top of it is redundant.